### PR TITLE
Add throw_on_user_data_attribute_failure parameter

### PR DESCRIPTION
**Pull request by RomulusED69**
see [ISSUE-11](https://github.com/gordalina/GordalinaMixpanelBundle/issues/11)

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -54,6 +54,10 @@ public function getConfigTreeBuilder()
                     ->info('Send data user on each master request (useful if you do not force users to disconnect when setting up the bundle or add new properties in their profile. WARNING: preferred used @Mixpanel\UpdateUser() or @Mixpanel\Set() at connection for performances)')
                     ->defaultValue($this->debug)
                 ->end()
+                ->booleanNode('user_data_display_errors')
+                    ->info('Throw Exception if user attribute not exist')
```
Comment (tristanbes) [inline DependencyInjection/Configuration.php]:
```suggestion
                    ->info('Throw Exception if user attribute does not exist')
```

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -54,6 +54,10 @@ public function getConfigTreeBuilder()
                     ->info('Send data user on each master request (useful if you do not force users to disconnect when setting up the bundle or add new properties in their profile. WARNING: preferred used @Mixpanel\UpdateUser() or @Mixpanel\Set() at connection for performances)')
                     ->defaultValue($this->debug)
                 ->end()
+                ->booleanNode('user_data_display_errors')
```
Comment (gordalina) [inline DependencyInjection/Configuration.php]: Can we name this field as: `throw_on_user_data_attribute_failure`? It's more verbose but conveys better what it refers to.

**Review by gordalina — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -332,6 +332,7 @@ gordalina_mixpanel:
     enabled: true                                 # defaults to true
     enable_profiler: %kernel.debug%               # defaults to %kernel.debug%
     auto_update_user: %kernel.debug%              # defaults to %kernel.debug%
+    user_data_display_errors: %kernel.debug%      # defaults to %kernel.debug%
```
Comment (gordalina) [inline README.md]: Missing renaming this field and the PR is good to go.

Comment (RomulusED69) [reply README.md]: oups indeed, thanks

**Review by gordalina — APPROVED** [review]

**Pull request merged**